app/enterprise/changelog.md

```diff
@@ -4,6 +4,167 @@ no_search: true
 no_version: true
 skip_read_time: true
 ---
+## 2.3.0.0
+**Release Date** 2020/02/10
+
+### **Kong Gateway (Enterprise)**
+The following sections list {{site.ee_product_name}}-exclusive updates,
+features, and fixes for the **2.3.0.0** version.
+
+Kong Enterprise is renamed to Kong Gateway, or Kong Gateway (Enterprise) going forward.
```

Comment:
I think a reader may read this and think that their "Kong Enterprise subscription is renamed to Kong Gateway subscription."  I think we need to be clear on what's changing (how we refer to the gateway name) and it makes sense to me to specifically call out the inverse (OSS) as well for completeness.  I've tried to identify some longer-form description in my blog which could be summarized here

Reply:
Add this to changelog "Kong Enterprise is renamed to Kong Gateway or Kong Gateway (Enterprise) going forward. For a more detailed explanation, including information about Kong Gateway (OSS), see [Kong Gateway (Enterprise) 2.3.x Release Notes](https://docs.konghq.com/enterprise/2.3.x/release-notes/). "

If we want a more detailed explanation in the changelog, we can expand.